### PR TITLE
[Tooling] Using credix pre-mainnet instead of devnet for CI

### PR DESCRIPTION
**Pull request by uxd-vincent**
Moving to latest credix testing program: pre-mainnet
Also removing tests for `collectProfitsFromCredix` and `redeemFromCredix` since those two IX are now replaced by the rebalancing logic instead and wont work in the latest mainnet

### CODE REVIEW

File: programs/uxd/tests/integration_tests/api/program_credix/binaries/dump.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 solana config set --url https://api.devnet.solana.com
-solana program dump crdszSnZQu7j36KfsMJ4VEmMUTJgrNYXwoPVHUANpAu executable-devnet.so
+solana program dump crdRi38zEhQdzpsxnKur73WHBM9BSvXMSfGcbLyJCdP executable-devnet.so
```
Comment (cnek) [inline programs/uxd/tests/integration_tests/api/program_credix/binaries/dump.sh:2]: call it executable-pre-mainnet?

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "@project-serum/anchor": "0.26.0",
     "@solana/spl-token": "0.3.7",
     "@solana/web3.js": "1.73.0",
-    "@uxd-protocol/uxd-client": "7.3.1-rc1"
+    "@uxd-protocol/uxd-client": "7.2.0-rc2"
```
Comment (cnek) [inline package.json:7]: should it stay `7.3.1-rc1` as discussed?

File: programs/uxd/tests/integration_tests/api/program_credix/binaries/dump.sh (hunk 1 above)
Comment (uxd-vincent) [reply programs/uxd/tests/integration_tests/api/program_credix/binaries/dump.sh:2]: Well, it does run on the devnet tho T_T

File: package.json (hunk 2 above)
Comment (uxd-vincent) [reply package.json:7]: for sure, will revert 👌

File: programs/uxd/tests/integration_tests/api/program_credix/binaries/dump.sh (hunk 1 above)
Comment (cnek) [reply programs/uxd/tests/integration_tests/api/program_credix/binaries/dump.sh]: yea let's keep the name then coz it's the only legit devnet program we should use going forward

**Review by acamill — DISMISSED** [review]

**Review by cnek — APPROVED** [review]

**Pull request merged**